hbase-server/src/main/java/org/apache/hadoop/hbase/HDFSBlocksDistribution.java

```diff
@@ -199,10 +225,23 @@ public long getUniqueBlocksTotalWeight() {
    * @return the locality index of the given host
    */
   public float getBlockLocalityIndex(String host) {
+    return getBlockLocalityIndex(host, false);
+  }
+  /**
+   * return the locality index of a given host
+   * @param host the host name
+   * @param forSsd only calc ssd type
+   * @return the locality index of the given host
+   */
+  public float getBlockLocalityIndex(String host, boolean forSsd) {
     float localityIndex = 0;
     HostAndWeight hostAndWeight = this.hostAndWeights.get(host);
     if (hostAndWeight != null && uniqueBlocksTotalWeight != 0) {
-      localityIndex=(float)hostAndWeight.weight/(float)uniqueBlocksTotalWeight;
+      if (forSsd){
+        localityIndex=(float)hostAndWeight.weightForSsd/(float)uniqueBlocksTotalWeight;
+      }else {
```

Comment:
Please use style of the surrounding code. See how there is a space between ){ and between }else.... Also an empty line between methods.

Reply:
will fix.